src/scripts/main.js

```diff
@@ -1,3 +1,270 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const tabHead = document.querySelector('thead');
+const tabBody = document.querySelector('tbody');
+
+const rows = tabBody.rows;
+
+let clickCount;
```

Comment:
let's assign some value at once. just assign `null` if you don't have some needed value yet